test/cmd/admin.sh

```diff
@@ -23,7 +23,8 @@ USE_IMAGES=${USE_IMAGES:-$defaultimage}
 
 # Test admin manage-node operations
 [ "$(openshift admin manage-node --help 2>&1 | grep 'Manage nodes')" ]
-[ "$(oadm manage-node --selector='' --schedulable=true | grep --text 'Ready' | grep -v 'Sched')" ]
+# test-cmd only runs the master, not the node
+#[ "$(oadm manage-node --selector='' --schedulable=true | grep --text 'Ready' | grep -v 'Sched')" ]
```

Comment:
I assume this only works against the Node API object... create one in order to manipulate it?